src/events/normalEvents/messageReactionAdd.ts

```diff
@@ -0,0 +1,62 @@
+import { Event } from "../../utils/Event";
+import { config } from "../../config";
+import type { APIEmbed } from "discord-api-types/v9";
+import { Logger } from "../../utils/Logger";
+
+export const event = new Event(
+	"messageReactionAdd",
+	async (client, reaction, user) => {
+		try {
+			if (
+				!reaction.message.channel.isText() ||
+				reaction.message.channel.id !== config.logs.suggestions ||
+				reaction.message.id === "931837510408089622" ||
```

Comment:
Data about this message may be partial. Better to do these checks later
```suggestion
				reaction.message.id === "931837510408089622" ||
```